pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -467,6 +468,28 @@ func (c *Cluster) EnableServiceMonitor(activeDaemon string) error {
 	}
 	serviceMonitor.Spec.NamespaceSelector.MatchNames = []string{c.clusterInfo.Namespace}
 	serviceMonitor.Spec.Selector.MatchLabels = c.selectorLabels(activeDaemon)
+
+	// If the cephcluster is managed by another resource,
+	// add the name of that resource as a label on the ceph metrics.
+	if c.spec.Labels != nil {
+		monitoringLabels, ok := c.spec.Labels["monitoring"]
```

Comment:
nit on consolidating the check for `ok`:
```suggestion
		if monitoringLabels, ok := c.spec.Labels["monitoring"]; ok {
```

Reply:
Ack. I will update.